docs/developer/extending/apps/architecture/app-permissions.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: App permissions
+---
+
+[Permissions](developer/permissions.mdx) are a way to control what operations can be performed by an app.
+
+## Managing permissions
+
+When you install an app, Saleor grants it a set of permissions. The initial app permissions are drawn from [the app's manifest file](developer/extending/apps/architecture/manifest.mdx) and stored in the [`App`](api-reference/apps/objects/app.mdx) object under the `permissions` field.
+
+Permissions can also be modified after the app is installed. You can change them using the [appUpdate](api-reference/apps/mutations/app-update.mdx) mutation:
+
+```graphql
+mutation UpdateAppPermissions($id: ID!) {
+  appUpdate(id: $id, input: { permissions: [MANAGE_ORDERS] }) {
+    app {
+      permissions {
+        name
+        code
+      }
+    }
+  }
+}
+```
+
+## Using the app
+
+Apps are only visible for the users who have the `MANAGE_APPS` permission.
```

Comment:
Keep in mind that this is technically a bug and we will change this behavior cc @krzysztofwolski

Reply:
I am aware, I even created the bug ticket for it, but we should document it nevertheless